character/human/body/.code/rig/arms/ik_clavicle/ik_clavicle.py

```diff
@@ -78,21 +78,19 @@ def main():
         
         #fix pole vector
         loc = cmds.spaceLocator(n = 'clavicle_rotateLock_%s' % side)[0]
+        locXform = cmds.group(loc ,n = 'xform_clavicle_rotateLock_%s' % side)

```

Comment:
locXform should follow the python pep8 rules
something like loc_xform instead of camelCase